Rakefile

```diff
@@ -22,3 +22,31 @@ Rake::TestTask.new(:test) do |t|
 end
 
 task default: :test
+
+desc "Deploy from a running instance. "
+task :deploy, %i(stack revision) => :environment do |_t, args|
+  begin
+    args.with_defaults(stack: nil, revision: nil)
+
+    stack = args[:stack]
+    revision = args[:revision]
+
+    raise ArgumentError.new('The first argument has to be a stack, e.g. shopify/shipit/production') if stack.nil?
+    raise ArgumentError.new('The second argument has to be a revision') if revision.nil?
+
+    module Shipit
+      class Task
+        def write(text)
+          p text
+          chunks.create!(text: text)
+        end
+      end
+    end
```

Comment:
Monkey path the write method so we can output the logs as they happen to the CLI. This monkey patch only affect this rake task.

Reply:
hm this is definitely a smell.

What if we add a property to `tasks` called `inline`/`skip_queue` that we pass here:
https://github.com/Shopify/shipit-engine/pull/895/files#diff-866796f172b542adeaf3b08f1d403511R128
so that we can just add a check to tasks#write instead? 😄

Reply:
I would prefer the job to be unaware of `inline`/`skip_queue`. The primary reason is that I don't want to introduce complexity that isn't necessary as `inline` is only going to be used for this command. If we had more use-cases for that option I would go that route.